Casks/g/geekbench-ai.rb

```diff
@@ -0,0 +1,25 @@
+cask "geekbench-ai" do
+  version "1.0.0,101711"
+  sha256 "122543b555ecfb4eb3b46b59e282cbb153e0f06639419f6fda3d7ce7231ff91e"
+
+  url "https://cdn.geekbench.com/GeekbenchAI-#{version.csv.first}-Mac.zip"
+  name "Geekbench AI"
+  desc "Cross-platform AI benchmark to evaluate AI workload performance"
+  homepage "https://www.geekbench.com/ai/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
+  end
+
+  auto_updates true
+  depends_on macos: ">= :ventura"
+
+  app "Geekbench AI.app"
+
+  zap trash: [
+    "~/Library/Caches/com.primatelabs.macos.geekbenchai",
+    "~/Library/HTTPStorages/com.primatelabs.macos.geekbenchai",
+    "~/Library/Preferences/com.primatelabs.macos.geekbenchai.plist",
```

Comment:
```suggestion
    "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/com.primatelabs.macos.geekbenchml.sfl*",
    "~/Library/Caches/com.primatelabs.macos.geekbenchml",
    "~/Library/HTTPStorages/com.primatelabs.macos.geekbenchml",
    "~/Library/Preferences/com.primatelabs.macos.geekbenchml.plist",
    "~/Library/Saved Application State/com.primatelabs.macos.geekbenchml.savedState",
```

The leftover files I found were named `geekbenchml` instead of `geekbenchai`. However, I assume they repurposed the code for the `geekbench-ml` app, so this might change.